hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/client/impl/TestBlockReaderLocal.java

```diff
@@ -130,10 +130,14 @@ public void doTest(BlockReaderLocal reader, byte original[])
         throws IOException {
       // default: no-op
     }
-  }
+    public void doTest(BlockReaderLocal reader, byte[] original, int shift)
+            throws IOException {
+      // default: no-op
+    }  }
 
   public void runBlockReaderLocalTest(BlockReaderLocalTest test,
-      boolean checksum, long readahead) throws IOException {
+      boolean checksum, long readahead, int shortCircuitCachesNum)
```

Comment:
A lot of test code changes are not needed if you add an overriding method runBlockReaderLocalTest(BlockReaderLocalTest test, boolean checksum, long readahead) that calls runBlockReaderLocalTest(test, checksum, readahead, 1);

Reply:
Thank you, I fixed it. But below by link writes "Could not update commit status". Should I create new PR or it is enough?